docs/adrs/00007-sbom-dashbord.md

```diff
@@ -0,0 +1,65 @@
+# 00007. Sbom dashboard.
+
+Date: 2025-04-27
+
+## Status
+
+DRAFT
+
+## Context
+This ADR document is intended to define and implement the backend logic for the heard of SBOM dashboard in the UI.
+
+Its mockup design document is as follows.
+![header-dashbord.png](header-dashbord.png)
+### This dashboard's header can be divided into three parts:
+- sbom state
+![sbom-status.png](sbom-status.png)
+  The information includes two components: the total sum of the Packages and the total sum of individual licenses (with Policy Violations removed). 
+- Vulnerabilities state
+![v11y-state.png](v11y-state.png)
+  It also contains the total number of Vulnerabilities, as well as the count for each severity level.
+- License info
+![sbom-info.png](sbom-info.png)
+  It consists of two parts:
+    - SBOM information (name, version, published date).
+    - License grouping - aggregating all unique licenses and visualizing them in a pie chart.
+
+## Decision
+Design an endpoint for each of these parts.
+- sbom state
+  -  **HTTP GET api//v2/sbom/{id}/sbom-status**
+  -  Reponse playload
+  ```json
+  {
+    "total_packages": "0",
+    "total_licenses": "0"
+  }
+  ```
+- Vulnerabilities state
+  -  **HTTP GET api//v2/sbom/{id}/vulnerabilities-status**
+  - -  Reponse playload
+  ```json
+  {
+    "total_vulnerabilities": "0",
+    "total_critical": "0",
+    "total_high": "0",
+    "total_medium": "0",
+    "total_low": "0"
```

Comment:
What about "none"?

Reply:
To my understanding `none` is a dedicated severity. So there can 5 vulnerabilities of type `none`.

Reply:
you are right.